doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -345,6 +464,9 @@ Simulation in a Notebook
         client_resources=backend_config["client_resources"],
     )
 
+For more advanced information regarding Flower simulation please read the
+|flower_how_to_run_simulations_link|_ guide.
```

Comment:
I'm thinking we could add this in a more prominent way (maybe with `.. tip:`?) at the beginning of both, CLI and notebook simulation sections?

Reply:
Nice suggestion. Added a small tip just below the different Flower versioning bullet points.